Engine/Source/Runtime/Rendering/WorldRenderer.cpp

```diff
@@ -180,8 +178,7 @@ void WorldRenderer::Render(float deltaTime)
 	{
 		MaterialComponent* pMaterialComponent = m_pCurrentSceneWorld->GetMaterialComponent(entity);
 		if (!pMaterialComponent ||
-			(pMaterialComponent->GetMaterialType() == m_pCurrentSceneWorld->GetPBRMaterialType() && pMaterialComponent->GetMaterialType() == m_pCurrentSceneWorld->GetCelluloidMaterialType()) ||
```

Comment:
@VTui22 
Cartoon renderer seems broken by this.
Is the cartoon renderer based on the mesh submitted by WorldRenderer? :sweat_drops:

Reply:
yes, celluloidrenderer just submit some parameters

Reply:
Reverted for now, please optimize it later.